packages/utils/src/metricsTimezone.js

```diff
@@ -0,0 +1,23 @@
+// @ts-check
+
+/**
+ * @returns {{
+ *   timezone: string,
+ *   gmt: number|string,
+ * }}
+ */
+const getTimezone = () => {
+  const dateParts = new Date().toString().split(' ');
+  const timezoneType = dateParts[dateParts.length - 2];
+  const timezoneOffset = dateParts[dateParts.length - 1];
+
+  const timezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+  const gmt = `${timezoneType} ${timezoneOffset}`;
```

Comment:
this code is not working. try to run this part in browser, and you'll see it that `gmt` value is bad.
all you need is:
```
const date = new Date();
return date.getTimezoneOffset();
```

Reply:
@ifiri or what format should be?

Reply:
It is not ready yet. I will call you